gammapy/modeling/models/temporal.py

```diff
@@ -79,6 +80,8 @@ def plot(self, time_range, ax=None, **kwargs):
             times to plot the model
         ax : `~matplotlib.axes.Axes`, optional
             Axis to plot on
+        nbin : int
```

Comment:
In `SpectralModel.plot` we called this `n_points`, so we are inconsistent here. I think the best solution would be to rename to `nbin` consistently, but that's backwards incompatible. The 2nd best solution would probably to at least go for consistency in the `.plot` methods an call this `n_points` here as well.

Reply:
modified to `n_points`